.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Report a bug in the Python client library
+about: Report a bug
+labels: bug
+---
+
+## Bug description
+
+Briefly describe the bug.
+
+## Version
+
+Which version of the library are you running?
+
+- Version:
+
+## IOTA network
+
+Which node are you connected to and which IOTA network is it in?
+
+- Node URL:
+- Network:
+
+## Hardware specification
+
+What hardware are you using?
+
+- Operating system:
+- RAM:
+- Number of cores:
+- Device type:
```

Comment:
❔ Are these needed most of the time?  We might remove them, so that people have fewer steps to complete.

Reply:
Other client library devs have said that it's useful to know which node you are connected to. I suppose some bugs may be related to unsynchronized nodes, bugs in the node software, or a problem with the network such as with the Coordinator.

As for the hardware spec, I'll leave that to you guys to decide 😄

Reply:
For the hardware specs, I think operating system is important. For example, the `pow` extension tends to have problems with windows or mac bc of the included binary.